test/unit/bin/expand-file-globs-test.js

```diff
@@ -23,15 +23,14 @@ describe('expandFileGlobs', function () {
       project.write({ 'application.hbs': 'almost empty' });
 
       let files = expandFileGlobs(['application.hbs', 'other.hbs'], []);
-      expect(files.has('application.hbs')).toBe(true);
-      expect(files.has('other.hbs')).toBe(false);
+      expect(files).toEqual(new Set(['application.hbs']));
```

Comment:
These assertion changes are making the test a bit stricter (matching on exact expected contents).